crates/storage/nippy-jar/src/writer.rs

```diff
@@ -119,6 +119,7 @@ impl<'a, H: NippyJarHeader> NippyJarWriter<'a, H> {
         // changes if necessary.
         if !is_created {
             writer.check_consistency_and_heal()?;
+            writer.commit()?;
```

Comment:
unrelated fix?

Reply:
yes, didn't actually faced any problems, but noticed that we should commit after a potential heal